test/controllers/gobierto_common/api/v1/search_controller_test.rb

```diff
@@ -40,6 +40,10 @@ def pending_event
           @pending_event ||= gobierto_calendars_events(:richard_pending)
         end
 
+        def madrid_dataset
```

Comment:
I don't like to have madrid in a var name, what about site_dataset or just dataset?

Also, review the testing part in the Rails boo. This is a fixture and it should be used like `gobierto_data_datasets(:users_dataset)` i.e.:

```
        def dataset
          @dataset ||= gobierto_data_datasets(:users_dataset)
        end
```